rbac/server/api/roles.py

```diff
@@ -66,6 +67,42 @@
 
 
 @ROLES_BP.get("api/roles")
+@doc.summary("Return a list containing all roles.")
+@doc.description("Return a list containing all roles.")
+@doc.consumes({"head": str}, location="query")
+@doc.consumes({"start": int}, location="query")
+@doc.consumes({"limit": int}, location="query")
+@doc.produces(
+    {
+        "data": [
```

Comment:
Data is missing a few fields like description and remote_id. Here's a sample payload from this API endpoint:
```
"data": [
    {
      "remote_id": "",
      "members": [
        "651b8eef-036a-453e-b05f-e54b65beea30"
      ],
      "description": "",
      "id": "68f93e35-f91b-48e1-89bb-f4357bc05e50",
      "created_date": 1565045223,
      "start_block_num": 2,
      "metadata": {
        "sync_direction": "OUTBOUND"
      },
      "tasks": [],
      "name": "NextAdmins",
      "administrators": [
        "651b8eef-036a-453e-b05f-e54b65beea30"
      ],
      "owners": [
        "651b8eef-036a-453e-b05f-e54b65beea30"
      ],
      "proposals": [],
      "end_block_num": 9223372036854776000,
      "packs": [
        "523c390f-ff74-4e37-860b-7f744e450d23"
      ]
    }
  ],
```